services/topics/config.js

```diff
@@ -0,0 +1,9 @@
+module.export = [
```

Comment:
There are no other `module.export`s in the repo so I'm wondering if there is a different standard.

Reply:
only if this was "type": "module".  Trying to minimize changes and get this to core sync.